src/actions/popup-action.js

```diff
@@ -0,0 +1,14 @@
+import * as types from './action-types';
```

Comment:
Also, it's quite common pattern to use camelCase in names of files instead of a hyphen.

Reply:
fixed